build/yamls/windows/containerd-with-ovs/containerd-with-ovs.yml

```diff
@@ -18,8 +18,10 @@ spec:
           image: antrea-windows
           imagePullPolicy: IfNotPresent
           args:
-            - -file
-            - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/Run-AntreaOVS-Containerd.ps1
+          - -command
+          - |
+            Move-Item -Path "$env:CONTAINER_SANDBOX_MOUNT_POINT/start_ovs_windows_containerd.ps1" -Destination "$env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/start_ovs_windows_containerd.ps1" -Force
```

Comment:
Is it possible to avoid this copy?

Reply:
I have added this move , because on keeping any other directory  other than /var/lib/antrea-windows  as the working directory of the script, I was getting 'permission denied' issue while moving the openvswitch directory from container to host. This issue does not appear in this case as /var/lib/antrea-windows is one of the mounted volumes.

Reply:
Can you explain what is the 'permission denied' issue?

Reply:
For example if we keep the script in the k directory and then run it , I get this error
`Copying OVS package from C:\C\caf5b5b031afeceb2cab7f33ed0178384277419d22905e6ce0f919cf9e2df301\k\openvswitch to C:\openvswitch
mv : Access to the path 'C:\C\caf5b5b031afeceb2cab7f33ed0178384277419d22905e6ce0f919cf9e2df301\k\openvswitch' is 
denied.`
This is true for any directory other than /var/lib/antrea-windows whic is one of the the volume mounts.

Reply:
This is not being used now